src/components/Feed/FeedCard.jsx

```diff
@@ -32,7 +32,11 @@ const FeedCard = ({
     className="post"
     key={slug}
   >
-    <div className="header">
+    <div
+      role="presentation"
+      className="header"
+      onClick={(e) => navigateTo(e, { push, url: `/posts/${slug}` })}
```

Comment:
Also, I might be confused here, why aren't we using Link to redirect ?

Reply:
@dusmel For styling purposes, and it provides more functionalities.

Reply:
I don't know but treating divs like anchors is not always a good idea. And a crime for accessibility 😉